### PR TITLE
bower version upgraded

### PR DESCRIPTION
**Pull request by PoetsRock**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "ll-token-field",
-  "version": "1.0.1",
+  "version": "1.2.0",
   "authors": [
-    "Anonymous <anonymous@example.com>"
+    "Aaron Hickman <ahickman@leisurelink.com>"
```
Comment (311chaos) [inline bower.json]: Zack actually authored this component.

**Comment by 311chaos** [comment]
Strange, it looks like this includes all of the changes. Any idea why it has so many changes? I thought these already went to master?

**Comment by PoetsRock** [comment]
@311chaos
1. cool... i'll update to zach ...  
2. yeah, i don't know about that other commit ... I guess it never got merged? looks like it was pushed 8 days ago.

**Comment by 311chaos** [comment]
It looks to me like it is already in master

**Comment by PoetsRock** [comment]
weird. ... then i don't know what happened.  ... can we kill that one commit?

**Pull request closed without merging**